program.txt

```diff
@@ -0,0 +1,11 @@
+define void factorial(int n)
+int fact =1;
+for(i=i;i<=n;i++)
```

Comment:
Please learn first google coding best practices thereafter you will let know yourself how to write code.